src/basic/glyph-util.c

```diff
@@ -129,6 +130,7 @@ const char *special_glyph(SpecialGlyph code) {
                         [SPECIAL_GLYPH_RECYCLING]               = u8"♻️",        /* actually called: UNIVERSAL RECYCLNG SYMBOL */
                         [SPECIAL_GLYPH_DOWNLOAD]                = u8"⤵️",        /* actually called: RIGHT ARROW CURVING DOWN */
                         [SPECIAL_GLYPH_SPARKLES]                = u8"✨",
+                        [SPECIAL_GLYPH_LOW_POWER]               = u8"⚡",
                         [SPECIAL_GLYPH_WARNING_SIGN]            = u8"⚠️",
                 },
         };
```

Comment:
please also update ./build/test-locale-util with a line for testing this new emoji